### PR TITLE
feat: Add .editorconfig

### PR DESCRIPTION
**Pull request by jachym-tousek-keboola**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.go]
+indent_style = space
+indent_size = 4
```
Comment (michaljurecko) [inline .editorconfig:9]: Go uses tabs, and it is standard:
https://stackoverflow.com/questions/19094704/indentation-in-go-tabs-or-spaces

Comment (jachym-tousek-keboola) [reply .editorconfig]: Ofc. :man_facepalming: Sorry, fixed.

**Review by michaljurecko — COMMENTED** [review]
LGTM

**Comment by jachym-tousek-keboola** [comment]
@michaljurecko you say LGTM but didn't actually click approval so I can't merge it. :D

**Review by michaljurecko — APPROVED** [review]
LGTM

(sorry)

**Pull request merged**